src/JsonUtil.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Emsifa\Laci;
+
+class JsonUtil 
+{
+    public static function load( $filename ) {
+        $contents = @file_get_contents( $filename );
+        if ( $contents === false ) {
+            return false;
+        }
+        return json_decode(self::stripComments($contents),true);
```

Comment:
`json_decode` fails decoding JSON with string contains "//"

![gambar](https://user-images.githubusercontent.com/6297931/117724369-d7d03e80-b20d-11eb-95c1-fc3611e2f921.png)

Reply:
Halo mas , untuk strip comments ini hanya komentar yang didalam object json bukan didalam string.

Reply:
Sebetulnya ga masalah ga bisa strip komentar, soalnya komentar itu bukan bagian dari RFC JSON. Juga file JSON disini konteksnya bukan untuk dibaca manusia. Kalaupun bisa strip komentar, komentar itu bakal hilang pas simpan perubahan di database (file JSON tsb).

https://github.com/emsifa/laci-db/blob/01d78dbe275d727ce27129aac850fde742df2505/src/Collection.php#L431

Reply:
> Sebetulnya ga masalah ga bisa strip komentar, soalnya komentar itu bukan bagian dari RFC JSON. Juga file JSON disini konteksnya bukan untuk dibaca dan ditulis manusia. Kalaupun bisa strip komentar, komentar itu bakal hilang pas simpan perubahan di database (file JSON tsb).
> 
> https://github.com/emsifa/laci-db/blob/01d78dbe275d727ce27129aac850fde742df2505/src/Collection.php#L431-L440

Oh i get it thanks ilmu baru, saya akan revert changes soal json ini, thank you mas, awesome.